File: services/dbservice/src/main/java/com/torin/dbService/dto/GiftsDto.java
package com.torin.dbService.dto;

import java.time.Instant;

public record GiftsDto(
        Long idGroup,
        Long idFrom,
        String idGift,
        String message,
        String titleGift,
        String flags,
        String flags2,
        Long stars,
        Long convertStars,
        Integer availabilityTotal,
        Instant date
) {
}

File: services/telegramservice/DTOs/DTO/GiftsDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DTOs.DTO
{
    public class GiftsDTO
    {
        public long? IdGroup { get; set; }
        public long? IdFrom { get; set; }
        public string? IdGift { get; set; }
        public string? Message { get; set; }
        public string? TitleGift { get; set; }
        public string? Flags { get; set; }
        public string? Flags2 { get; set; }
        public long? Stars { get; set; }
        public int? AvailabilityTotal { get; set; }
        public long? ConvertStars { get; set; }
        public DateTime? Date { get; set; }
    }
}